### PR TITLE
[xml] Update italian.xml

### PR DESCRIPTION
**Pull request by conky77**
* [ce58b42](https://github.com/notepad-plus-plus/notepad-plus-plus/commit/ce58b424bdeec3616943f9ea8cd257f66700c3c0) Add Pin tab feature]
* [b822480](https://github.com/notepad-plus-plus/notepad-plus-plus/commit/b8224808bdb6f9eaf94c066219634715703828da) Add ability to open/copy selected files from Search-results]
* [eeb4753](https://github.com/notepad-plus-plus/notepad-plus-plus/commit/eeb47530449cebabc9c94a8bccefc0f574a2491c) Add "Minimize / Close to" option for System tray]

### CODE REVIEW

File: PowerEditor/installer/nativeLang/italian.xml [hunk 1]
```diff
@@ -1723,9 +1725,9 @@ Cerca in tutti i file, ma escludi tutte le cartelle o sottocartelle log o logs:
 			<finder-collapse-all value="Riduci tutto"/>
 			<finder-uncollapse-all value="Espandi tutto"/>
 			<finder-copy value="Copia righe selezionate"/>
-			<finder-copy-paths value="Copia Percorso/i"/>
+			<finder-copy-paths value="Copia Percorso/i selezionati"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/italian.xml:1728]: Need to change

File: PowerEditor/installer/nativeLang/italian.xml [hunk 2]
```diff
@@ -1723,9 +1725,9 @@ Cerca in tutti i file, ma escludi tutte le cartelle o sottocartelle log o logs:
...
 			<finder-clear-all value="Pulisci tutto"/>
-			<finder-open-all value="Apri tutto"/>
+			<finder-open-all value="Apri tutti quelli selezionati"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/italian.xml:1730]: Need to change

**Review by donho — CHANGES_REQUESTED** [review]
Need to change for aligning with the master

**Comment by conky77** [comment]
Since I wasn't in time, I created a new Pull Request

**Pull request closed without merging**